xds/src/main/java/io/grpc/xds/internal/sds/ServerWrapperForXds.java

```diff
@@ -77,27 +132,112 @@ public Server start() throws IOException {
       throw new RuntimeException(ex);
     }
     delegate.start();
```

Comment:
If here throws a recoverable exception, should it retry as well?

Reply:
For example if there is a BindException? If you look at this [spec](https://github.com/grpc/proposal/blob/master/A36-xds-for-servers.md#serving-and-not-serving) it allows for this scenario i.e. "If the server is unable to open serving ports (e.g., because the port is already in use by another process), XdsServer startup may fail" and I prefer this approach since it's backward compatible. Existing code that expects `start()` to either succeed or fail immediately.